src/core/cpuid.d

```diff
@@ -344,7 +341,7 @@ CpuFeatures cpuFeatures;
  */
 CpuFeatures* getCpuFeatures() @nogc nothrow
 {
-    pragma(inline, false);
+    version (GNU) {} else pragma(inline, false);
```

Comment:
The version of GDC mentioned above does not support pragma(inline, ...) unless you disable unknown pragmas. Does the comment above the function apply to GCC in the same way? I could use you noinline attribute from gcc.attributes.